wremnants/plot_tools.py

```diff
@@ -1794,7 +1799,7 @@ def make_summary_plot(
             loc=logoPos,
             lumi=lumi,
             no_energy=lumi is None,
-            text_size=textsize,
+            text_size=get_textsize(ax1, "small"),
```

Comment:
Is this desired? Previously the textsize of the cms label was the same as the size of the legend, avoiding too many different font sizes which gives a more harmonized plot.

Reply:
There was a request to make CMS bigger. I think it's not unreasonable to have it stand out

Reply:
But this change doesn't make the CMS bigger, "small" means it keeps it always at the size of the label ticks. Or better to say it keeps what is followed by the CMS to this size, the CMS is bigger by a factor of 1.2 or so anyway. I would adapt that in a future PR again if you agree